pkg/router/template/template_helper.go

```diff
@@ -173,25 +148,57 @@ func endpointsForAlias(alias ServiceAliasConfig, svc ServiceUnit) []Endpoint {
 	return endpoints
 }
 
-// sortedMapData returns the sorted data in a haproxy map. The returned data is
-// in alphabetically reverse order. The sortSubGroups option sorts the
-// non-wildcard paths first and adds the sorted wildcard paths at the end.
-// Note: The reversed sorting ensures we do a longest path match first.
-func sortedMapData(name string, sortSubGroups bool) []string {
-	lines, err := fileutil.ReadLines(name)
-	if err != nil {
-		glog.Errorf("Error reading map file %s: %v", name, err)
-		return []string{}
+// Returns a haproxy backend config for a given service alias.
```

Comment:
i would keep the godoc standard even if this is private func

Reply:
ok will do. thx